### PR TITLE
[Makefile] Add first_line_match for makefile shebang

### PR DESCRIPTION
**Pull request by dlech**

### CODE REVIEW

File: Makefile/Makefile.sublime-syntax [hunk 1]
```diff
@@ -12,6 +12,7 @@ file_extensions:
   - OCamlMakefile
   - mak
   - mk
+first_line_match: '^#!/usr/bin/(make|env\s+make)'
```
Comment (wbond) [inline Makefile/Makefile.sublime-syntax]: This should probably be something more like: `'^#!/usr/bin/(make|env\s+make)\b'` to ensure `make` isn't part of a longer word.

**Comment by wbond** [comment]
Do people not use the `/usr/bin/env make` format?

**Comment by dlech** [comment]
I don't think so. I found a StackOverflow question about this and the conclusion was no. I also tried it and it didn't work because you have to pass the `-f` command line option to the `make` command itself. So, I'm pretty sure no one uses it.

**Comment by wbond** [comment]
The only other thought would be to match `/usr/local/bin/make`, although that seems unlikely in practice. I also don't imagine distros ship `make` as `/bin/make`.

**Comment by dlech** [comment]
Well, we can always change it if we find out there is another case.

If you think this is good enough for now, I can squash if you don't use the squash merge feature of GitHub.

**Pull request merged**